src/OSPSuite.Infrastructure.Import/Core/Mappers/DataSetToDataRepositoryMapper.cs

```diff
@@ -151,6 +152,15 @@ private void convertParsedDataColumn(DataRepository dataRepository, KeyValuePair
          dataRepository.Add(dataColumn);
       }
 
+      private double truncateUsingLLOQ(SimulationPoint value)
+      {
+         if (value == null) return double.NaN;
```

Comment:
Let's not reutrn on the same line
I was hoping that Resharper file formatting should catch this.
so 
if(value==null)
  return double.NaN;

Reply:
I need to check if I still have Resharper working with the file formatting. I know I installed VisualStudio Community (not preview anymore) at some point and not sure if I updated everything. I will check today. Thanks for the heads up

Reply:
Return at new line is done now